core/src/main/java/org/apache/iceberg/TableMetadata.java

```diff
@@ -208,8 +210,9 @@ public String toString() {
     this.lastUpdatedMillis = lastUpdatedMillis;
     this.lastColumnId = lastColumnId;
     this.schema = schema;
-    this.specs = specs;
     this.defaultSpecId = defaultSpecId;
+    this.specs = specs;
+
```

Comment:
Can you revert these unnecessary changes? The order of assignments changed and there's an extra newline. These non-functional changes can cause git conflicts, so we try to avoid them.